### ISSUE TITLE
Should save to session storage. Not local storage

### ISSUE THREAD

**Issue by jaxoncreed** [inrupt/solid-client-authn-js#474]
Again, this is leftover from my code. So, sorry about that.

Currently the library saves things like the `code_verifier` to localstorage. It should be session storage instead:

https://github.com/inrupt/solid-client-authn-js/blob/master/packages/browser/src/storage/BrowserStorage.ts#L34-L40

**Comment by NSeydoux** [comment]
Good point. The local storage is cleared on login/logout, so this would not really be an issue, but it would absolutely be cleaner to have this in the session storage.

**Comment by pmcb55** [comment]
Yeah, thanks again Jackson - we're tracking this internally now, so we'll close this issue once we're resolved it.
